src/utils/module.c

```diff
@@ -70,12 +71,16 @@ gboolean bd_utils_have_kernel_module (const gchar *module_name, GError **error)
     }
 
     path = kmod_module_get_path (mod);
-    have_path = (path != NULL) && (g_strcmp0 (path, "") != 0);
+    module_present = (path != NULL) && (g_strcmp0 (path, "") != 0);
+    if (!module_present) {
+      int state = kmod_module_get_initstate(mod);
```

Comment:
Please move the `state` definition to the start of the function (and also use `gint` instead of `int` please).

Reply:
Also please add space after the function name.